src/plugin.tsx

```diff
@@ -53,6 +53,35 @@ export const pluginStandalone: BackstagePlugin = createPlugin({
 });
 
 export function getEntityMock(): Entity {
+  return {
```

Comment:
Also do these need to be functions? nit-pic so no big deal.

Reply:
They are functions just because i don't want them to be at the beginning of the file. After move to separate file with mocks it'll be just exported objects.